weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CompareEvaluationsPage/sections/ComparisonDefinitionSection/EvaluationDefinition.tsx

```diff
@@ -46,8 +46,9 @@ export const EvaluationCallLink: React.FC<{
   state: EvaluationComparisonState;
 }> = props => {
   const evaluationCall = props.state.data.evaluationCalls[props.callId];
-  const [entity, project] =
-    evaluationCall._rawEvaluationTraceData.project_id.split('/');
+  const entity = props.state.data.entity;
```

Comment:
Could be `const {entity, project} = props.state.data;`?

Reply:
👍 in next commit